runners/spark/src/main/java/org/apache/beam/runners/spark/stateful/SparkGroupAlsoByWindowViaWindowSet.java

```diff
@@ -97,7 +97,8 @@
   private static final Logger LOG =
       LoggerFactory.getLogger(SparkGroupAlsoByWindowViaWindowSet.class);
 
-  private static class StateAndTimers implements Serializable {
+  /** State and Timers wrapper. */
+  public static class StateAndTimers implements Serializable {
```

Comment:
does the class still need to be serializable?

Reply:
not valid anymore. Only added classes for kryo registration, default is still JavaSerializer